source/guide_passbolt.rst

```diff
@@ -108,20 +120,114 @@ Install the dependencies:
 ::
 
  [isabell@stardust ~]$ cd ~/html/
- [isabell@stardust html]$ wget --output-document=composer.phar https://getcomposer.org/composer-1.phar
+ [isabell@stardust html]$ wget --output-document=composer.phar https://getcomposer.org/composer.phar
  [isabell@stardust html]$ php composer.phar install --no-dev
  [isabell@stardust html]$ rm composer.phar
- [isabell@stardust html]$ cp config/passbolt.default.php config/passbolt.php
+
```

Comment:
Is there no `config/passbolt.default.php` anymore in the package? Do you need to create the config completely from scratch with this new version?

Reply:
No, there is still a `config/passbold.default.php` provided.
I dumped it, because u still need to edit the config manually, and it's spammed with comments.
See: https://github.com/passbolt/passbolt_api/blob/master/config/passbolt.default.php

To get started and have an easier config, I added a minimalistic version with config vars in the header. That would simplify the configuration and search/replace process.

If someone needs more or extended configurations, I will assume that he will look into passbolt docs himself.

Reply:
We want our users to understand what they're doing and not just copy files without thinking. For us it makes more sense to use the provided file (which can change with newer versions) and change thee needed settings.